pkg/registry/pod/etcd/etcd_test.go

```diff
@@ -641,26 +608,27 @@ func TestEtcdUpdateScheduled(t *testing.T) {
 			SecurityContext:               &api.PodSecurityContext{},
 		},
 	}
-	_, _, err := storage.Update(ctx, &podIn)
+	_, _, err = storage.Update(ctx, &podIn)
 	if err != nil {
 		t.Errorf("Unexpected error: %v", err)
 	}
-	response, err := fakeClient.Get(key, false, false)
+
+	obj, err := storage.Get(ctx, "foo")
 	if err != nil {
-		t.Fatalf("Unexpected error: %v", err)
+		t.Errorf("Unexpected error: %v", err)
 	}
-	var podOut api.Pod
-	testapi.Default.Codec().DecodeInto([]byte(response.Node.Value), &podOut)
-	if !api.Semantic.DeepEqual(podOut, podIn) {
-		t.Errorf("expected: %#v, got: %#v", podOut, podIn)
+	podOut := obj.(*api.Pod)
+	// can not do full comparison b/c we are adding all the meta fields now
```

Comment:
I meant:
"Can you please explicitly say that validChangedPod() is changing only Labels, so we just compare this?"